platform/src/test/java/platformfeatures/PF_Header.feature

```diff
@@ -0,0 +1,13 @@
+Feature: To check Header Links
+
+@FunctionalTest
+Scenario Outline: Validate Header Links
+
+Given Navigate to URL and open Header <browser>
```

Comment:
Seperate opening and closing browser into different steps

Reply:
Done Changes as requested

Reply:
Feature file needs more details:
Given open browser
And navigate to url
When -user performs action
And -user performs another action-
Then -validate test result-
And close browser

Please refer online resources for feature file writing in cucumber

Reply:
Done Changes as requested